.github/workflows/main.yaml

```diff
@@ -18,4 +18,11 @@ jobs:
         uses: actions/checkout@v3
       - name: Run the linter
         run: make lint
-
+  format:
+    name: fmt
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Run the format check
+        run: make fmt check-uncommitted
```

Comment:
You can configure the linter to catch this.
https://github.com/kiagnose/kiagnose/blob/main/.golangci.yml

Reply:
which linter specifically? I can't find it

Reply:
The linter will fail if the code is not properly formatted.

Reply:
Like! But adding this config doesn't suit this PR very well.
I will experiment adding this lint config in another PR and then remove this check. OK?

Reply:
> Like! But adding this config doesn't suit this PR very well. 

I agree

Reply:
Ok looks good enough. 
```
$ make lint
docker run --rm \
           --volume `pwd`:/home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup:Z \
           --workdir /home/ralavi/go/src/github.com/kiagnose/kubevirt-dpdk-checkup \
            docker.io/golangci/golangci-lint:v1.50.1 golangci-lint run --timeout 3m ./cmd/...
cmd/main.go:25: File is not `gofmt`-ed with `-s` (gofmt)
	fmt.Println("kubevirt-dpdk-checkup starting..."   )
make: *** [Makefile:45: lint] Error 1
```
will issue the PR

Reply:
https://github.com/kiagnose/kubevirt-dpdk-checkup/pull/8

Reply:
Removed the fmt job